### PR TITLE
Wallet tutorial heading update

### PR DESCRIPTION
**Pull request by pyr0gan**
- Add top headings to all pages
- Add emojis to all headings

### CODE REVIEW

**Review by dgamboa — APPROVED** [review]
Looks great, solid emoji choices!

File: figment-learn/new-pathways/solana-wallet/wallet-tutorial-intro.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
+# 🚀 Introduction 
+
 You're about to learn how to develop a wallet for the [Solana](https://solana.com/) protocol. As you build, we'll dive into some of the fundamentals of Web 3 so you can start cultivating an understanding for blockchain technology and decentralized applications (dApps).
 
-# Prerequisites
+# 📋 Prerequisites
```
Comment (dgamboa) [inline figment-learn/new-pathways/solana-wallet/wallet-tutorial-intro.md:5]: Can we make these ##?

File: figment-learn/new-pathways/solana-wallet/wallet-tutorial-step-2.md [hunk 2]
```diff
@@ -18,7 +20,7 @@ Run and open the app in your browser at [http://localhost:3000](http://localhost
 
 Clicking on **Create New Wallet** routes the user to a `/generate` page signaling to our user that by clicking **Generate**, the app will generate a key phrase and thus create a new wallet. But when we click **Generate**, we're routed to a `/phrase` page with an empty phrase container.
 
-# Implementation
+# 🧩 Implementation
```
Comment (dgamboa) [inline figment-learn/new-pathways/solana-wallet/wallet-tutorial-step-2.md:23]: Same here, let's make these ##

**Review by dgamboa — CHANGES_REQUESTED** [review]

File: figment-learn/new-pathways/solana-wallet/wallet-tutorial-step-2.md (hunk 2 above)
Comment (pyr0gan) [reply figment-learn/new-pathways/solana-wallet/wallet-tutorial-step-2.md]: Got it 👍

**Review by dgamboa — APPROVED** [review]
Now it's perfecto

**Pull request merged**